### PR TITLE
bmeck topics for March

### PR DESCRIPTION
**Pull request by bmeck**

### CODE REVIEW

File: 2018/03.md [hunk 1]
```diff
@@ -110,9 +110,13 @@ Supporting materials includes slides, a link to the proposal repository, a link
         1. Update on Frozen Realms in light of Meltdown and Spectre (Mark S. Miller)
         1. Meeting host volunteers for early 2019 (Daniel Ehrenberg)
         1. Getting last item of Array for stage 2 (Keith Cirkel) ([explainer](https://github.com/keithamus/proposal-array-last), [spec](https://github.com/keithamus/proposal-array-last#specification))
+        1. Debugger operands for stage 1 ([github](https://github.com/bmeck/proposal-debugger-operands))
     1. 30-minute items
         1. Extended Numeric Literals (Daniel Ehrenberg) ([slides](https://docs.google.com/presentation/d/1fEhTnHGyIBIHBiImT6W0eYdBLLY0oRGyjAq6dItZgqE/edit#slide=id.p), [explainer](https://github.com/tc39/proposal-extended-numeric-literals/blob/master/README.md), [spec](https://tc39.github.io/proposal-extended-numeric-literals/))
+        1. Richer Keys for stage 1 ([slides](https://docs.google.com/presentation/d/1q3CGeXqskL1gHTATH_VE9Dhj0VGTIAOzJ1cR0dYqDBk/edit#slide=id.p))
```
Comment (jridgewell) [inline 2018/03.md]: Need share access.

Comment (bmeck) [reply 2018/03.md]: woops, fixed?

Comment (jridgewell) [reply 2018/03.md]: Yup.

**Comment by littledan** [comment]
@bmeck You should have write access to the agendas repository; is that not working for you?

**Comment by bmeck** [comment]
@littledan I never just merge my own stuff. I generally let things sit. There isn't really a process described for how we should be PR'ing these things.

**Comment by ljharb** [comment]
Prior to the deadline, a PR isn’t really necessary on this repo; it never hurts to get review, but I’d say feel free to push directly to master if you like.

**Pull request merged**